src/gradualizer_db.erl

```diff
@@ -344,19 +344,18 @@ import_module_from_erl(Mod, State) ->
 import_erl_files([File | Files], State) ->
     EppOpts = [{includes, guess_include_dirs(File)}],
     {ok, Forms} = epp:parse_file(File, EppOpts),
-    [{attribute, _, file, _} | Forms1] = Forms,
```

Comment:
Good.

What I think we should use these for is when code is `-include`'d, in the error messages. We should keep track of in which file each piece of code is.